### PR TITLE
allow ingress

### PR DESCRIPTION
**Pull request by whites11**
the cilium network policy does not have any ingress allowed, meaning the api server can't connect to metrics server.

### CODE REVIEW

File: helm/metrics-server-app/templates/ciliumnetworkpolicy.yaml [hunk 1]
```diff
@@ -14,4 +14,7 @@ spec:
     - toEntities:
         - kube-apiserver
         - cluster
+  ingress:
```
Comment (QuentinBisson) [inline helm/metrics-server-app/templates/ciliumnetworkpolicy.yaml:17]: Why is that needed ? 🤔

Comment (whites11) [reply helm/metrics-server-app/templates/ciliumnetworkpolicy.yaml:17]: a message to your team's channel with a detailed explanation is coming.

Comment (QuentinBisson) [reply helm/metrics-server-app/templates/ciliumnetworkpolicy.yaml:17]: You're scaring me :D

I think we never added ingress because no component should scrape that but I guess the api-server is?

**Review by QuentinBisson — APPROVED** [review]
